### PR TITLE
- Set sip_calculation screen in root of project

### PR DESCRIPTION
**Pull request by Parthshiroya06**
- I have to set sip calculation screen in root of project. as @bharathkumarchowdary  said to me.

### CODE REVIEW

File: app/sipCalculator.tsx [hunk 1]
```diff
@@ -0,0 +1,348 @@
+import { useAuthContext } from '@/hooks/use-auth'
```
Comment (rockingrohit9639) [inline app/sipCalculator.tsx:1]: Please follow file naming conventions.
The file name should be `sip-calculator.tsx`

File: app/sipCalculator.tsx [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+    },
+  })
+
+  const summery = calculateSip(Number(watch('sip')), Number(watch('year')), Number(watch('aunual_return')))
+
+  const navLogin = () => {
+    router.push('/login')
+  }
+  const summaryText = (title: string, value: string | number) => {
+    return (
```
Comment (rockingrohit9639) [inline app/sipCalculator.tsx:42]: Please make them proper components

File: app/sipCalculator.tsx [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+import { PieChart } from 'react-native-chart-kit'
+export default function SipCalculator() {
+  const { width } = useWindowDimensions()
+  const router = useRouter()
+  const navig = useNavigation()
+
+  const { tokenData } = useAuthContext()
+
+  const { handleSubmit, register, control, setValue, getValues, watch } = useForm({
+    defaultValues: {
```
Comment (rockingrohit9639) [inline app/sipCalculator.tsx:29]: Instead of using `useForm` you can store data directly in states.. that will be more convinient and fast as well

**Review by rockingrohit9639 — COMMENTED** [review]
Please follow theme colors

File: app/sipCalculator.tsx (hunk 2 above)
Comment (Parthshiroya06) [reply app/sipCalculator.tsx]: Hello, @rockingrohit9639  bhai, This feedback is resolve.

File: app/sipCalculator.tsx (hunk 3 above)
Comment (Parthshiroya06) [reply app/sipCalculator.tsx]: Hello @rockingrohit9639 Bhai ,
-  I have to solve laggy issue using  react -  native state component. 
- Could you please verify that if any issue please let me know that

**Pull request closed without merging**